client/extensions/woocommerce/app/promotions/promotion-create.js

```diff
@@ -3,39 +3,124 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Main from 'components/main';
```

Comment:
We try to keep the imports alphabetized, these are slightly out of order.

Reply:
I’m a little confused about our alphabetization strategy for imports in .js files. Do we alphabetize by the symbols we import? Or the file names from which we import?